### PR TITLE
test hydrologie_hintergrundkarte

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

**Comment by pfanguin** [comment]
This is an aggregate (wms-bgdi_dev), which is in prod for the next deploy.

File: chsdi/models/vector/bafu.py [hunk 1]
```diff
@@ -11,6 +11,19 @@
 Base = bases['bafu']
 
 
+class hintergrundkarte(Base, Vector):
```
Comment (procrastinatio) [inline chsdi/models/vector/bafu.py]: Class name begins with UPPERCASE

**Comment by procrastinatio** [comment]
Minor error, but orhterwise LGTM

File: chsdi/models/vector/bafu.py (hunk 1 above)
Comment (AFoletti) [inline chsdi/models/vector/bafu.py]: That's not consistent at all... should we go through every model and change to uppercase every class?

**Pull request merged**